skpro/utils/set_output.py

```diff
@@ -0,0 +1,71 @@
+"""Utilities for set_output functionality."""
+
+__author__ = ["julian-fong"]
+
+from skpro.datatypes import convert
+from skpro.utils.validation._dependencies import _check_soft_dependencies
+
+SUPPORTED_OUTPUTS = ["pandas", "default"]
+
+if _check_soft_dependencies(["polars", "pyarrow"], severity="none"):
+    SUPPORTED_OUTPUTS.append("polars")
+
+
+SUPPORTED_OUTPUT_MAPPINGS = {
+    "pandas": ("pd_DataFrame_Table", "Table"),
+    "polars": ("polars_eager_table", "Table"),
+}
+
+
+def check_output_config(estimator):
```

Comment:
I would make this function and the next one private, to avoid user calls and allowing us to make changes quickly